runner.js

```diff
@@ -442,10 +450,10 @@ async function main () {
       }
       fix += chalk.bold(configPath)
     }
-    output += '\n  ' + chalk.yellow(fix)
+    hint = chalk.yellow(fix)
   }
 
-  process.stdout.write('\n' + output + '\n')
+  reporter.logResults({ results, hint })
```

Comment:
You do not need objects here. Just use arguments `reporter.logResults(results, hint)`

Reply:
Done in all methods